docs/.vuepress/theme/components/Footer.vue

```diff
@@ -1,82 +1,44 @@
 <template>
-  <footer class="footer">
-    <div class="credits">
-      Artwork by
-      <a
-        href="https://impermanence.co"
-        target="_blank"
-        rel="noopener noreferrer"
-        >Lili Feyerabend</a
-      >
-      feat.
-      <a href="https://ilankatin.com" target="_blank" rel="noopener noreferrer"
-        >ilan katin</a
-      >,
-      <a
-        href="https://linktr.ee/mattiacprodukt"
-        target="_blank"
-        rel="noopener noreferrer"
-        >Mattia Cuttini</a
-      >,
-      <a
-        href="https://oficinastk.github.io"
-        target="_blank"
-        rel="noopener noreferrer"
-        >Oficinas TK</a
-      >,
-      <a href="https://xcopyart.com" target="_blank" rel="noopener noreferrer"
-        >XCOPY</a
-      >.
+  <footer class="footer" id="footer">
+    <div class="top-row">
+      <p class="updated-date">{{ lastUpdatedText }}: {{ lastUpdatedDate }}</p>
```

Comment:
Can we add the icon & "Ethereum.org" text here like in Figma?
<img width="391" alt="Image 2020-02-28 at 12 15 27 PM" src="https://user-images.githubusercontent.com/8097623/75548038-2425c900-5a24-11ea-96db-e2c0e2fabde0.png">

Reply:
Done this, however I've skipped the "Ethereum.org" part as it's dependant on the translation which follows making sense